sample_plugin/modules/sequential_update.h

```diff
@@ -0,0 +1,39 @@
+#ifndef BESS_MODULES_SEQUENTIALUPDATE_H_
+#define BESS_MODULES_SEQUENTIALUPDATE_H_
+
+#include "module.h"
+#include "utils/endian.h"
+#include "utils/random.h"
+
+#include "protobuf/supdate_msg.pb.h"
+
+#define MAX_VARS 16
```

Comment:
minor: `static const size_t kMaxVariable` in `SequentialUpdate` declaration, instead of the macro